spec/features/reviews/user_views_reviews_spec.rb

```diff
@@ -29,4 +29,33 @@
     expect(page).to have_content(review.description)
     expect(page).to have_content(review.user.email)
   end
+
+  scenario 'visitor sees 25 reviews per page' do
+   neighborhood = FactoryGirl.create(:neighborhood)
+   25.times do |n|
+     FactoryGirl.create(
+       :review,
+       description: "Awesome!",
+       created_at: 5.days.ago,
+       neighborhood: neighborhood
+      )
+   end
+   25.times do |n|
+     FactoryGirl.create(:review,
+       description: "Great!",
+       neighborhood: neighborhood
+     )
+   end
+
+    visit neighAdAdborhood_path(neighborhood)
+
+    expect(page).to have_content("Great!")
+    expect(page).to_not have_content("Awesome!")
+
+
+    click_link("2")
+    expect(page).to have_content("Awesome!")
+    expect(page).to_not have_content("Great!")
+   end
```

Comment:
`end` at 59, 3 is not aligned with `scenario 'visitor sees 25 reviews per page' do` at 33, 2